examples/react/stores/src/index.tsx

```
@@ -0,0 +1,84 @@
+import ReactDOM from 'react-dom/client'
+import { type Store, useCreateStore, useSelector } from '@tanstack/react-store'
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)